setup.cfg

```diff
@@ -1,3 +1,15 @@
+[isort]
```

Comment:
isort is a tool that sorts the imports in Python files; or verifies that they are. I feel this is a worthwhile addition to pip since there have been multiple occasions where I've felt the need to sort the imports for neatness but avoided doing so to prevent excessive churn to the PR.